boot/core/src/boot/task/built_in.clj

```diff
@@ -174,11 +174,11 @@
             (throw t)))))))
 
 (core/deftask ^{:boot/from :jeluard/boot-notify} notify
-  "Aural and visual notifications during build.
+  "Audible and visual notifications during build.
 
-  Default audio themes: system (the default), ordinance, pillsbury,
-  and woodblock. New themes can be included via jar dependency with
-  the sound files as resources:
+  Default audio themes: system (the default), ordinance and
+  woodblock. New themes can be included via jar dependency with the
+  sound files as resources:
```

Comment:
Pillsbury is totally still a thing. I took it off the list because it was originally not listed and @alandipert mentioned that it was a boot easter egg.  I'm fine keeping it on the list or not, I guess I already spilled the secret :smile: